Jenkinsfile_CNP

```diff
@@ -10,6 +10,7 @@ def component = "general-applications"
 def camundaBranch = "master"
 def generalappCCDBranch = "sprint3integration"
 
+
```

Comment:
Could we revert this change if it's not intentional please?

Reply:
Yes we can do now